Maestro/subscriptions.json

```diff
@@ -142,54 +90,43 @@
             "/p:NotifyGitHubUsers=dotnet/coreclr-contrib",
```

Comment:
FYI - Jan just removed these NotifyGitHubUsers.  [https://github.com/dotnet/versions/pull/91](https://github.com/dotnet/versions/pull/91).  Make sure you merge this correctly.